docs/guided-tour/blueprints/helm-chart-resource/README.md

```diff
@@ -45,7 +45,7 @@ components:
           # notice that this has to be a reference to the updated blueprint
           imageReference: eu.gcr.io/gardener-project/landscaper/examples/blueprints/guided-tour/helm-chart-resource:1.0.0
       - name: helm-chart
-        type: helm.io/chart
+        type: helmChart
```

Comment:
I found several other places with "type: helm.io/chart" in the documentation. Have you also checked if these are still valid?